xml_test.go

```diff
@@ -0,0 +1,97 @@
+package vmap
+
+import (
+	"encoding/xml"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParsingSimple(t *testing.T) {
+	t.Parallel()
+	is := assert.New(t)
+	vmap := &VMAP{}
+	is.Nil(xml.Unmarshal([]byte(vmapPostRollBumper), vmap))
+	if !is.NotNil(vmap) {
+		t.FailNow()
+	}
+
+	is.Equal(2, len(vmap.AdBreak))
```

Comment:
throwing it out here as an idea: maybe add `msg` here explaining the intention of the assertion?